src/Uno.UI/UI/Xaml/Controls/Border/Border.Helpers.cs

```diff
@@ -33,29 +37,6 @@ private static Size HelperCollapseThickness(Thickness thickness)
 			return new Size(thickness.Left + thickness.Right, thickness.Top + thickness.Bottom);
 		}
 
-		internal static bool UseLayoutRoundingForBorderThickness(FrameworkElement element)
-		{
-			// Only snap BorderThickness if:
-			// 1) LayoutRounding is enabled - prerequisite for any pixel snapping.
-			// 2) There is no CornerRadius - we cannot guarantee snapping when there is a CornerRadius.
-			// 3) Plateau != 1.0 - this snapping is to ensure that integral values of BorderThickness don't cause subpixel rendering at high plateau.
-			// TODO: Remove check for plateau, BorderThickness should be consistently rounded at all plateaus.
-			//return (RootScale.GetRasterizationScaleForElement(element) != 1.0f) &&
-			//       element.GetUseLayoutRounding() &&
-			//       (!HasNonZeroCornerRadius(element.GetCornerRadius()));
-			return (RootScale.GetRasterizationScaleForElement(element) != 1.0f) &&
-				   element.GetUseLayoutRounding() &&
-				   (!HasNonZeroCornerRadius(element.GetCornerRadius()));
-		}
```

Comment:
This was a particularly sensitive behavior that was added, especially on iOS. We'll need to validate that this still works properly. Make sure to validate with related issues when blaming this function.

Reply:
@jeromelaban iOS shouldn't be affected. `GetUseLayoutRounding()` is always false for iOS.

https://github.com/unoplatform/uno/blob/58b7d854aacf85bc71a701040651a99c419d5615/src/Uno.UI/UI/Xaml/UIElement.cs#L1105-L1112

Reply:
Even previously the `GetUseLayoutRounding` was always `false` so this shouldn't change anything on non-Skia targets